test/e2e/node/pods.go

```diff
@@ -180,9 +180,9 @@ var _ = SIGDescribe("Pods Extended", func() {
 		/*
 			Release : v1.9
 			Testname: Pods, QOS
-			Description:  Create a Pod with CPU and Memory request and limits. Pos status MUST have QOSClass set to PodQOSGuaranteed.
+			Description:  Create a Pod with CPU and Memory request and limits. Pod status MUST have QOSClass set to PodQOSGuaranteed.
 		*/
-		framework.ConformanceIt("should be submitted and removed ", func() {
+		framework.ConformanceIt("creating pod with CPU and memory resource, pod QOS classification MUST be set PodQOSGuaranteed", func() {
```

Comment:
This phrasing  doesn't work.. `KubeDescribe(...) + It(...)` yields "Pods Set QOS Class creating pod with CPU and memory..." 

Maybe something like this, but... what is it that's actually causing this specific QOS class to be set?
```suggestion
		framework.ConformanceIt("should be set on Pods with resource requests and limits", func() {
```

Reply:
```
For a Pod to be given a QoS class of Guaranteed:

Every Container in the Pod must have a memory limit and a memory request, and they must be the same.
Every Container in the Pod must have a CPU limit and a CPU request, and they must be the same.    
```
https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed